jfrog_platform/README.md

```diff
@@ -71,10 +62,16 @@ The API key is configured in `td-agent`, which will start sending logs to Datado
 
 Add all attributes as facets from **Facets** > **Add** (on the left side of the screen in Logs) > **Search**.
 
-To access already existing visualizations and filters, click on Dashboards and add a new screenboard and then import [export.json][4] and overwrite the existing dashboard.
+### Step 3 Install JFrog Platform Tile 
+If you have not installed the JFrog Platform tile yet, install the tile.
+
+### Step 4 Open JFrog Artifactory Dashboard
+Go to Dashboard -> Dashboard List, find `JFrog Artifactory Dashboard` and open it.
+
+### Step 5 Search for logs
+Logs can be access in the UI under  **Logs** > **Search**. Select the specific source mentioned in the Fluentd configuration (`fluentd` in configuration example above) to retrieve logs.
 
 
 [1]: https://github.com/jfrog/charts
 [2]: https://helm.sh/
 [3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/jfrog_platform/images/dashboard.png
```

Comment:
```
[4]: https://app.datadoghq.com/account/settings#api
[5]: https://app.datadoghq.com/logs
```

Reply:
Hello there. I accepted all your changes. Only these two footer additions did not show up with "accept suggestion" button.

Reply:
@vinayagg - You will need to add them using your text editor

Reply:
@ruthnaebeck Thank you, I added these references.